lib/cdo.rb

```diff
@@ -118,6 +118,20 @@ def hourofcode_url(path = '', scheme = '')
       site_url('hourofcode.com', path, scheme)
     end
 
+    def javabuilder_url(path = '', scheme = '')
+      if (rack_env?(:development))
+        # Since pegasus and dashboard share the same port, we have a Route53
+        # DNS record that redirects requests to localhost. Javabuilder, as a
+        # separate service, uses a different port. Therefore, we can access the
+        # the service directly.
+        'ws://localhost:8080/javabuilder'
+      else
+        # TODO: Update to use this URL once we have Route53 set up for API Gateway
+        # site_url('javabuilder.code.org', '', 'wss')
+        'wss://nsgeezjyul.execute-api.us-east-1.amazonaws.com/staging'
```

Comment:
should this always be /staging?

Reply:
For now, yes, but we'll update this to whichever naming system we end up using.